app/frontend/src/components/forestrysectoropscreening/admin/AdminReviewSubmission.vue

```diff
@@ -41,21 +41,21 @@
 <script>
 import { mapGetters } from 'vuex';
 
-import Step2 from '@/components/minesoperatorscreening/Step2.vue';
-import Step3 from '@/components/minesoperatorscreening/Step3.vue';
-import Step4 from '@/components/minesoperatorscreening/Step4.vue';
-import Step5 from '@/components/minesoperatorscreening/Step5.vue';
+import Step1 from '@/components/forestrysectoropscreening/Step1.vue';
+import Step2 from '@/components/forestrysectoropscreening/Step2.vue';
+import Step3 from '@/components/forestrysectoropscreening/Step3.vue';
+import Step4 from '@/components/forestrysectoropscreening/Step4.vue';
 
 export default {
-  name: 'ReviewSubmission',
+  name: 'AdminReviewSubmission',
   components: {
+    Step1,
     Step2,
     Step3,
-    Step4,
-    Step5
+    Step4
   },
   computed: {
-    ...mapGetters('form', ['gettingForm', 'attestation', 'submissionComplete']),
+    ...mapGetters('ForestrySectorOpScreeningForm', ['gettingForm', 'attestation', 'submissionComplete']),
```

Comment:
This needs to be `forestrySectorOpScreeningForm` (note the lowercase f. This namespace is loaded on the top level forestrysectoropscreening.vue file and should match that.

Reply:
oh ok. thanks!